.travis.yml

```diff
@@ -1,10 +1,5 @@
-language: python
```

Comment:
so, your answer is no, it is not sensible to remove it. But for whatever reason it is removed, we don't know if @archetipo  has done it on purpose.